.aspect/cli/config.yaml

```diff
@@ -2,5 +2,5 @@ configure:
   languages:
     javascript: true
     go: true
-    kotlin: true
+    kotlin: false
```

Comment:
IDW why kotlin gazelle decided to fail with this refactor but simple thing to do is to disable it here